cmd/admin-rpc-client.go

```diff
@@ -39,6 +40,7 @@ type adminCmdRunner interface {
 	Restart() error
 	ListLocks(bucket, prefix string, duration time.Duration) ([]VolumeLockInfo, error)
 	ReInitDisks() error
+	Uptime() (time.Time, error)
```

Comment:
> @vadmeste this can be done with time.Duration too. I happen to pick time.Time for the comparisons in a distributed setup to compute the uptime of the distributed setup. I have taken uptime to be the duration for which the distributed setup has been available w.r.t read operations.

In that case, I think this API should be called BootTime, but also I really think that we should use duration instead, since duration will be correctly calculated starting from go 1.9 , otherwise you'll need to be sure that the system's time is correct.